csharp/extractor/Semmle.Extraction/Options.cs

```diff
@@ -20,6 +20,36 @@ public abstract class CommonOptions : ICommandLineOptions
         /// </summary>
         public Verbosity LegacyVerbosity { get; protected set; } = Verbosity.Info;
```

Comment:
Can this be made private?

Reply:
Unit tests are still accessing this property, so it can't be made `private`.